cli/man/splinter-database-migrate.1.md

```diff
@@ -0,0 +1,61 @@
+% SPLINTER-DATABASE-MIGRATE(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter-database-migrate** — Updates the Biome database for a new Splinter
+release
+
+SYNOPSIS
+========
+
+**splinter database migrate** \[**FLAGS**\] \[**OPTIONS**\]
+
+DESCRIPTION
+===========
+
+Biome is the Splinter module that manages users, credentials, and private keys,
+using a PostgreSQL database to store this information.
+
+This command migrates the Biome database from one Splinter release to the next.
+If a new release adds new database tables or changes existing table formats,
+run this command to update the Biome database to the new format.
+
+The "database-migrate-biome" Cargo feature must be enabled in order to update
```

Comment:
Remove this comment, it is temporary information.

Reply:
Deleted.